src/CommunityToolkit.Maui.UnitTests/Converters/IsListNullOrEmptyConverter_Tests.cs

```diff
@@ -16,21 +16,21 @@ public class ListIsNullOrEmptyConverter_Tests : BaseTest
 
 	[Theory]
 	[MemberData(nameof(Data))]
-	public void ListIsNullOrEmptyConverter(object value, bool expectedResult)
+	public void IsListNullOrEmptyConverter(object value, bool expectedResult)
 	{
-		var listIstNullOrEmptyConverter = new ListIsNullOrEmptyConverter();
+		var listIstNullOrEmptyConverter = new IsListNullOrEmptyConverter();
 
-		var result = (bool)listIstNullOrEmptyConverter.Convert(value, typeof(ListIsNullOrEmptyConverter), null, CultureInfo.CurrentCulture);
+		var result = (bool)listIstNullOrEmptyConverter.Convert(value, typeof(IsListNullOrEmptyConverter), null, CultureInfo.CurrentCulture);
 
-		Assert.Equal(result, expectedResult);
+		Assert.Equal(expectedResult, result);
 	}
 
 	[Theory]
 	[InlineData(0)]
 	public void InValidConverterValuesThrowArgumenException(object value)
```

Comment:
Such a minor point but

```suggestion
	public void InvalidConverterValuesThrowArgumenException(object value)
```